__tests__/apiTopics.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+describe("Get / api/topics", () => {
+  it("returns with array of objects in topics", () => {
+    return request(app)
+      .get("/api/topics")
```

Comment:
you can chain on a  `.expect(200)` here instead of needing line 19 i.e.

`return request(app).get("/api/topics").expect(200).then((response)....etc....`